sound/soc/sof/intel/hda-stream.c

```diff
@@ -497,6 +497,9 @@ int hda_dsp_stream_init(struct snd_sof_dev *sdev)
 	num_playback = (gcap >> 12) & 0x0f;
 	num_total = num_playback + num_capture;
 
+	sdev->num_playback = num_playback;
+	sdev->num_capture = num_capture;
```

Comment:
what happens if we have a platform with HDaudio and DMICs? They are already on the market for WHL windows platforms, need to think of this case upfront.

Reply:
It is removed